pokemon-checker/src/components/PokemonDisplay/PokemonDisplay.tsx

```diff
@@ -21,31 +21,39 @@ type displayProps = {
 
 export const PokemonDisplay = (props: displayProps) => {
   const [pokeonObject, setPokemonObject] = useState<PokemonDTO>();
-  //TODO (jeremy): Move this factory to a service! Views shouldn't control this.
-  const pokemonFactory: PokemonFactory = new PokemonFactory();
-
-  useEffect(() => {
-    fetchPokemonObject();
-  });
+  // TODO (jeremy): Elminate the factory declaration! The service method is
+  // passed to this component as a prop.
+  const pokemonFactory: PokemonFactory = useMemo(
```

Comment:
The [useMemo](https://usehooks.com/useMemo/) hook allows us to save the result of a function, so the component can remember what value it is every time it renders.

I included it because ESLint was yelling at me.